core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineMysql.java

```diff
@@ -309,4 +310,25 @@ public Driver getDriver() {
       throw new RuntimeException(e);
     }
   }
+
+  @Override
+  public String getPattern(LikeExpression likeExpression) {
+    String escape = likeExpression.getEscape();
+    String pattern = likeExpression.getTextValue();
+    if (escape.isEmpty()) {
+      // MySQL accepts an empty escape character to disable the escape function, but MariaDB ignores
```

Comment:
We need a tricky conversion for MySQL and MariaDB to handle both in the same manner since we don't want to distinguish them at this moment. But, in the future, we probably support the MariaDB driver and introduce `RdbEngineMariadb` or something like that.

Reply:
This is just an idea, but how about giving up disabling escape on ScalarDB side like MariaDB does? In this way, users would need to use the default escape or specify a single escape character, but can't specify an empty string.

Reply:
Thank you for the idea. That's one of the options I thought. However, all other databases other than MariaDB support disabling the escape, and it's probably convenient for users. So, can we go with it as is?